src/util.ts

```diff
@@ -997,6 +1010,21 @@ export function isTokenDetectionSupportedForNetwork(chainId: string): boolean {
   );
 }
 
+/**
+ * Check if token list polling is enabled for a given network.
+ * Currently this method is used to support e2e testing for consumers of this package.
+ *
+ * @param chainId - ChainID of network
+ * @returns Whether the current network supports tokenlists
+ */
+export function isTokenListSupportedForNetwork(chainId: string): boolean {
+  const chainIdDecimal = convertHexToDecimal(chainId).toString();
```

Comment:
this method needs to be able handle chainIds passed as hexadecimal since it is called in the [`fetchTokenMetadata` ](https://github.com/MetaMask/controllers/pull/855/files#diff-5b65de1cdf2aa82d2d39d41c5e52958a03bb5c66ccbd22d5cf60feb8005ae938R74)which is itself called in the `TokensController`, which in the context of the extension represents the `chainId` as a hex string.